### PR TITLE
post for oxford workshop

### PR DESCRIPTION
**Pull request by damianodamiani**
with a future date of 14 05, meaning it wont appear in the local built

### CODE REVIEW

**Comment by LenaHenningsen** [comment]
Thanks a lot. Can you correct the typo in colleagues please?
Good to merge in my view, at whatever date ;) 14th is fine

**Comment by damianodamiani** [comment]
> I fixed the typo. Plus, to run "bundle exec jekyll serve **--future**" instead of "bundle exec jekyll serve" can show the future post(s)

woow thats nice! Ive been waiting for that for a long time :) OKi so ready to merge, if you could

**Review by whalekeykeeper — APPROVED** [review]

**Comment by whalekeykeeper** [comment]
> > I fixed the typo. Plus, to run "bundle exec jekyll serve **--future**" instead of "bundle exec jekyll serve" can show the future post(s)
> 
> woow thats nice! Ive been waiting for that for a long time :) OKi so ready to merge, if you could

Done. Yay~

**Pull request merged**
